scripts/io_stability/collect-k8s-stats.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+DIR=$1
+mkdir -p "$DIR"
```

Comment:
what if the dir already exists ?
Does the script throw error and exit ?

Reply:
No, even if dir exists, there will be no error